finality-aleph/src/nodes.rs

```diff
@@ -149,7 +149,11 @@ where
         genesis_header,
     );
     let finalizer = AlephFinalizer::new(client.clone(), metrics.clone());
-    let (sync_service, justifications_for_sync, _) = match SyncService::new(
+    let (sync_service, justifications_for_sync, _): (
+        SyncService<aleph_primitives::Block, _, _, _, _, _, _>,
```

Comment:
Probably better to write this on the right hand side, by `SyncService` before `::new`. This should disappear after we actually use the `_` above though, right?